Tutorial.ipynb

```diff
@@ -2,7 +2,7 @@
  "cells": [
   {
    "cell_type": "code",
-   "execution_count": 1,
+   "execution_count": 32,
```

Comment:
The changes here are mostly the result of me rerunning the notebook to check it all worked using GWDM v1.2 in the API.  It ran without errors (except where errors were expected).